pkgs/applications/editors/emacs/elisp-packages/yes-no/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation {
```

Comment:
Use `trivialPackage` to package Emacs code.

Reply:
You mean `trivialBuild`?